dp/generate_prsn.py

```diff
@@ -222,6 +229,11 @@ def process_to_prsn(variables, output_dataset, max_chunk_len):
             }
         means = np.mean([chunk_data['tasmin'], chunk_data['tasmax']], axis=0)
         prsn_data = np.where(means < freezing, chunk_data['pr'], 0)
+
+        prsn_units_from = ureg.parse_units('kg/m^2/s')
+        prsn_units_to = ureg.parse_units('g/cm^2/s')
```

Comment:
Can you tell me more about using this as the target unit? You've changed both the mass (kg -> g) and the area (per square meter to per square centimeter). Is that definitely what we want? And is it definitely correct?

Reply:
Converting from kg -> g multiplies the data by 1000 and converting from /m^2 -> /cm^2 divides the data by 10000, giving the net effect of dividing the data by 10. Since converting from mm to cm also divides by 10, I think this will work. I can check with Lee when he's back.